mibidata/panels.py

```diff
@@ -57,10 +57,20 @@ def read_csv(path):
         # Combine and convert 'Mass' column to int
         combined = pd.concat(df, ignore_index=True)
         combined['Mass'] = combined['Mass'].astype(np.int64)
-        return _merge_masses(combined)
+        return merge_masses(combined)
 
 
-def _merge_masses(df):
+def merge_masses(df):
+    """Merges 'Target' cells of a dataframe with the same 'Mass' value.
+
```

Comment:
The sentence above is a good one line summary of the function. For docstrings in our code, we follow the convention that the summary is a single line (as it is here) and when appropriate, additional details or a more long-winded description can be added after a blank line. For this docstring, could you add "This function merges multiple targets that are conjugated to the same mass tag such that the returned DataFrame contains only unique masses. Target names are combined using the conventions of `util.natural_sort()`." as additional details?

Reply:
Small addition: in the docstrings, you can also reference other functions, such as the `util.natural_sort()` here such that, when the docs are automatically created after your merge, one can click on this function and immediately go to the doc of that function. It pretty much creates a hyperlink.

Though I don't know how to do this to some function in another python file as in this case. I leave it up to you to figure it out ;-)

Reply:
For fear of you wasting time looking up how to do it, @murataksoy and I just figured out it's :func:`function_name` where you replace function_name with the function you are trying to link to.